README.md

```diff
@@ -61,231 +33,158 @@ For more information on the importance of a professional README for your reposit
+## 👥 Authors 
 
-<!-- FUTURE FEATURES -->
+👤 **Uwimbabazi Dorcas**
 
-## 🔭 Future Features <a name="future-features"></a>
+- GitHub: [@Dorcas126](https://github.com/Dorcas126)
+- Twitter: [@Dorcasuw](https://twitter.com/Dorcasuwi)
+- LinkedIn: [Uwimbabazi Dorcas](https://www.linkedin.com/in/uwimbabazi-dorcas-956a5a226/)
 
-> Describe 1 - 3 features you will add to the project.
+👤 **Victor Asum**
+- LinkedIn: victorasumdev
+- Github: @AsumVictor
+- Twitter: @iamasum369
+- IG: aiasumcode
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
 
+## :telescope: Future Features <a name="future-features"></a><br/>
+- [ ] **Adding BackEnd**<br/>
+- [ ] **Making it stylish**
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- CONTRIBUTING -->
-
-## 🤝 Contributing <a name="contributing"></a>
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
+## :handshake: Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!<br/>
+Feel free to check the [https://github.com/yin-ka/To-Do-list/issues](../../issues/).
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
-## ⭐️ Show your support <a name="support"></a>
-
-> Write a message to encourage readers to support your project
-
-If you like this project...
 
+## :star:️ Show your support <a name="support"></a>
+If you like this project please follow me on github & twitter and also connect on Linkedin.
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+- I would like to thank Microverse for this exercise. <br>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
-
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
 
-  - [Answer_1]
+## :question: FAQ (OPTIONAL) <a name="faq"></a><br/>
+- **How long can it take to build that project**
 
-- **[Question_2]**
-
-  - [Answer_2]
+  - four days according to personal skills
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+This project is [MIT](./LICENSE) licensed.
```

Comment:
The license file is not working and leads to an error page (404 page not found). Kindly correct it to give your project a more professional look. :wink:

Reply:
Thank you. You have included the license file.